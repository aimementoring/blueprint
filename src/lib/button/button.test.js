import React from 'react';
import Button from './button';
import renderer from 'react-test-renderer';

describe('Button', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(<Button type="button" onClickFunction={() => {}} text="Back" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
