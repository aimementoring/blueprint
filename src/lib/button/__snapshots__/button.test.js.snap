// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders properly 1`] = `
<div
  className="container "
>
  <button
    className=" button"
    onClick={[Function]}
    type="button"
  >
    Back
  </button>
</div>
`;
